include/dt-bindings/pinctrl/cc13xx_cc26xx-pinctrl.h

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright (c) 2015 - 2017, Texas Instruments Incorporated
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef CC13XX_CC26XX_PINCTRL_COMMON_H_
+#define CC13XX_CC26XX_PINCTRL_COMMON_H_
+
+/* Adapted from hal/ti/simplelink/source/ti/devices/cc13x2_cc26x2/driverlib/ioc.h */
+
+//*****************************************************************************
+//
+// IOC Peripheral Port Mapping
+//
+//*****************************************************************************
```

Comment:
please no C99 style comments

Reply:
this file is a trimmed down version from hal/ti/..driverlib/ioc.h , all styling issues fixed for this file as separate commit